ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -303,6 +314,12 @@ private VectorizedOrcAcidRowBatchReader(JobConf conf, OrcSplit orcSplit, Reporte
           VectorizedRowBatch.DEFAULT_SIZE, null, null, null);
     }
     rowIdProjected = areRowIdsProjected(rbCtx);
+    rowIsDeletedProjected = isVirtualColumnProjected(rbCtx, VirtualColumn.ROWISDELETED);
+    if (rowIsDeletedProjected) {
+      rowIsDeletedVector = new RowIsDeletedColumnVector();
```

Comment:
Lets explicitly pass VectorizedRowBatch.DEFAULT_SIZE to make this more obvious

Reply:
done